app/src/main/java/org/mozilla/focus/browser/LocalizedContent.java

```diff
@@ -110,6 +114,43 @@ private static void loadRights(@NonNull final WebView webView) {
         webView.loadDataWithBaseURL("file:///android_asset/rights.html", data, "text/html", "UTF-8", null);
     }
 
+    /**
+     * Load the content for focus:debug
+     */
+    private static void loadDebug(@NonNull final WebView webView) {
+        final Context context = webView.getContext();
+        final Resources resources = Locales.getLocalizedResources(context);
+
+        final Map<String, String> substitutionMap = new ArrayMap<>();
+        final String userAgentString = webView.getSettings().getUserAgentString();
```

Comment:
Let's add "User Agent: " preceding the string here to be clear what we're showing.